npm-packages/meteor-babel/package.json

```diff
@@ -53,6 +53,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-decorators": "7.14.5",
     "@babel/plugin-syntax-decorators": "7.14.5",
+    "@babel/plugin-transform-async-generator-functions": "^7.25.7",
```

Comment:
Given that `meteor-babel` depends on `babel-preset-meteor` where this dependency is coming from (and was changed in the past already) this might not be necessary.

Reply:
Hey, thanks for the input. Can you elaborate as to what I should be changing?

Reply:
I don't think this addition is needed. Why did you add it?

Reply:
I saw that the outdated plugin was present in dependencies of that folder, I uninstalled it and installed the asked dependency.

Reply:
That is not what the changelog is saying. This is listed as a pure addition. If you just run the commands that would make sense.

Reply:
Let's try removing it and see if things still work. They should as we are getting this dependency from `babel-preset-meteor`

Reply:
sure I will make the changes